fars/booking/templates/profile.html

```diff
@@ -0,0 +1,115 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% block css %}
+{% load static %}
+<link rel="stylesheet" href='{% static "css/profile.css" %}' />
+{% endblock %}
+
+{% block calendar %}
+<div class="row col-10 offset-1">
+  <div class="col-8">
+    <div class="row text-center">
+      <h3 class="col-12">{% trans "Your Bookings" %}</h3>
+    </div>
+    <!-- Ongoing bookings -->
+    <div class="row">
+      <h5 class="col-12">{% trans "Ongoing bookings" %}<hr></h5>
+      {% for b in ongoing_bookings %}
+      <div class="col-10 offset-1 profile-event">
+        <div class="row">
+          <div class="col-6 offset-1 font-weight-bold">
```

Comment:
This is probably supposed to be col-3

Reply:
Don't understand which line you mean

Reply:
Commented line, i.e. 20. I believe it should be same as line 28.

Reply:
Not relevant anymore then. Ongoing bookings now use the same design as future bookings

Reply:
I think I liked the previous design better :/
Not too fond of text-center